app-frontend/src/app/index.bootstrap.js

```diff
@@ -8,8 +8,22 @@ import './index.module';
 
 import '../assets/styles/sass/app.scss';
 
+import deferredBootstrapper from 'angular-deferred-bootstrap';
+
 angular.element(document).ready(function () {
-    angular.bootstrap(document, ['rasterFoundry'], {
-        strictDi: true
+    deferredBootstrapper.bootstrap({
+        element: document,
+        module: 'rasterFoundry',
+        bootstrapConfig: {
+            strictDi: true
+        },
+        resolve: {
+            APP_CONFIG: ['$http', ($http) => {

```

Comment:
guess I'll leave it as-is - the caps make it clear that it's not a normal angular service